dhctl/pkg/kubernetes/actions/deckhouse/install.go

```diff
@@ -307,11 +307,11 @@ func CreateDeckhouseManifests(kubeCl *client.KubernetesClient, cfg *Config) erro
 				return manifests.ClusterUUIDConfigMap(cfg.UUID)
 			},
 			CreateFunc: func(manifest interface{}) error {
-				_, err := kubeCl.CoreV1().ConfigMaps("kube-system").Create(context.TODO(), manifest.(*apiv1.ConfigMap), metav1.CreateOptions{})
+				_, err := kubeCl.CoreV1().ConfigMaps(manifests.ClusterUUIDCmNamespace).Create(context.TODO(), manifest.(*apiv1.ConfigMap), metav1.CreateOptions{})
 				return err
 			},
 			UpdateFunc: func(manifest interface{}) error {
-				_, err := kubeCl.CoreV1().ConfigMaps("kube-system").Update(context.TODO(), manifest.(*apiv1.ConfigMap), metav1.UpdateOptions{})
+				_, err := kubeCl.CoreV1().ConfigMaps(manifests.ClusterUUIDCmNamespace).Update(context.TODO(), manifest.(*apiv1.ConfigMap), metav1.UpdateOptions{})
```

Comment:
NIT: 

I personally do not like declaring constants with long names, when they are the part of an object. As for me, it better to get the name from an object.
```go
clusterUUIDCm := manifest.(*apiv1.ConfigMap)
fmt.Println(clusterUUIDCm.Name, clusterUUIDCm.Namespace)
```
or

```go
clusterUUIDCm := manifests.ClusterUUIDConfigMap()
fmt.Println(clusterUUIDCm.Name, clusterUUIDCm.Namespace)
```

Reply:
But is it not a blocker for this PR.